planb/PBManifest.m

```diff
@@ -135,6 +137,11 @@ - (BOOL)validatePackage:(id)object {
       return NO;
     }
   }
+  if (package[kPlanBIgnoreKey] && ![package[kPlanBIgnoreKey] isKindOfClass:[NSNumber class]]) {
```

Comment:
The "package[kPlanBIgnoreKey]" check is redundant. Sending a isKindOfClass: message to nil is safe and will return nil (NO).

Reply:
The first check is to see if the key exists -- it's optional so I don't want the second check to fail if there is no planb_ignore key in the manifest.

Reply:
Ah, gotcha